src/inference/src/dev/core_impl.cpp

```diff
@@ -232,10 +232,11 @@ ov::SoPtr<ov::ICompiledModel> import_compiled_model(const ov::Plugin& plugin,
     if (auto blob_hint = config.find(ov::hint::compiled_blob.name()); blob_hint != config.end()) {
         try {
             auto compiled_blob = blob_hint->second.as<ov::Tensor>();
-            ov::SharedStreamBuffer buffer{reinterpret_cast<char*>(compiled_blob.data()), compiled_blob.get_byte_size()};
-            std::istream stream{&buffer};
-            compiled_model =
-                context ? plugin.import_model(stream, context, config) : plugin.import_model(stream, config);
+            auto _config = config;
+            _config.erase(ov::hint::compiled_blob.name());
```

Comment:
Why is required, the hint should be meaning less for plugin if compiled blob passed as parameter

Reply:
There is no need to pass compiled_blob as parameter anymore. We can use new api instead

Reply:
The property should be ignored by device in this case there is no need to make copy and process the configuration.
Can it be updated.